src/js/edu-benefits/components/Introduction.jsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+
+class Introduction extends React.Component {
```

Comment:
In HCA we used the naming convention of adding `Section` to each section component to make it clear what that component was used for (e.g, `IntroductionSection`). This way we wouldn't confuse `Introduction`, which could be a blurb on a larger page component, with `IntroductionSection`, which is more clearly a page in and of itself. We could also add `Page` instead of `Section` to be super clear.